drizzlepac/devutils/comparison_tools/compare_sourcelists.py

```diff
@@ -300,135 +399,166 @@ def computeLinearStats(matchedRA,plotGen,diffMode,plot_title,plotfile_prefix,ver
-    log.info(">>>>>> Comparision - reference sourcelist {} differences <<<<<<".format(plot_title))
-    #remove any "inf" or "nan" values in matchedRA.
+    log.info(">>>>>> Comparision - reference sourcelist {} absolute differences <<<<<<".format(plot_title))
+    # remove any "inf" or "nan" values in matchedRA.
     nanIdx = np.where(np.isnan(matchedRA) == True)[1]
-    if len(nanIdx) >0:
-        if verbose: log.info("{} np.nan values will be removed from input list. New input list contains {} values.".format(len(nanIdx),len(matchedRA[0,:]) - len(nanIdx)))
+    if len(nanIdx) > 0:
+        if verbose:
+            log.info("{} np.nan values will be removed from input list. New input list contains {} values.".format(len(nanIdx), len(matchedRA[0, :]) - len(nanIdx)))
         matchedRA = np.delete(matchedRA, nanIdx, axis=1)
 
     infIdx = np.where(np.isinf(matchedRA) == True)[1]
-    if len(infIdx) >0:
-        if verbose: log.info("{} np.inf values will be removed from input list. New input list contains {} values.".format(len(infIdx),len(matchedRA[0,:]) - len(infIdx)))
+    if len(infIdx) > 0:
+        if verbose:
+            log.info("{} np.inf values will be removed from input list. New input list contains {} values.".format(len(infIdx), len(matchedRA[0, :]) - len(infIdx)))
         matchedRA = np.delete(matchedRA, infIdx, axis=1)
 
     # 'sigma' and 'iters' input values used for various np.sigma_clipped_stats() runs
-    sigVal= 3
-    intersVal= 3
-
-    if diffMode == "absolute":
-        diffRA=matchedRA[1, :] - matchedRA[0, :] # simple difference
-
-    else:
-        if diffMode == "pmean":
-            normValue = np.abs(sigma_clipped_stats(matchedRA[0, :], sigma=sigVal, maxiters=intersVal)[0]) #divide all comp-ref values by a single sigma-clipped mean ref value
-            if verbose: log.info("normValue: {}".format(normValue))
-        if diffMode == "pdynamic":
-            normValue = matchedRA[0, :] #divide each comp-ref value by the corresponding ref value
-
-        diffRA = ((matchedRA[1, :] - matchedRA[0, :]) / normValue) * 100.0
+    sigVal = 3
+    intersVal = 3
 
+    diffRA = matchedRA[1, :] - matchedRA[0, :]  # simple difference
     clippedStats = sigma_clipped_stats(diffRA, sigma=sigVal, maxiters=intersVal)
     pct_1sig = (float(np.shape(np.where(abs(diffRA) <= clippedStats[2]))[1]) / float(np.shape(diffRA)[0])) * 100.0
+    sigma_percentages = []
+    for sig_val in [1.0, 2.0, 3.0]:
+        sigma_percentages.append((float(np.shape(np.where((diffRA >= (clippedStats[0] - sig_val * clippedStats[2])) & (diffRA <= (clippedStats[0] + sig_val * clippedStats[2]))))[1])/float(np.shape(diffRA)[0])) * 100.0)
+    #pct_3sig = ((float(np.shape(np.where(diffRA <= 3.0*clippedStats[2] + clippedStats[0]))[1]) + float(np.shape(np.where(diffRA >= clippedStats[0] - 3.0 * clippedStats[2]))[1]))/float(np.shape(diffRA)[0])) * 100.0
     pct_five = (float(np.shape(np.where(abs(diffRA) <= 5.0))[1]) / float(np.shape(diffRA)[0])) * 100.0
```

Comment:
This computes the statistics on each set of values.  However, regardless of whether it is being provided RA in arcseconds or X in pixels or FLUX1 in counts, this line will always compare the difference to a constant value of 5.0.  I would suggest that this value be provided as an input to this routine to allow the calling routine to determine what limit is appropriate for the given values and their units.